lib/models/BucketInfo.ts

```diff
@@ -144,6 +144,7 @@ export default class BucketInfo {
         objectLockEnabled?: boolean,
         objectLockConfiguration?: any,
         notificationConfiguration?: any,
+        tags?: { key: string; value: string }[] | null,
```

Comment:
Ok one more point on this line, I would use a TS type or whatever its called instead of building it each time.

Reply:
I created the `BucketTag` type here: https://github.com/scality/Arsenal/commit/b964834d3135e98184538a8176b0b5630c8c147b#diff-7aa7eeaed88643ea97d57a5f15ddc2e64c380190641c53bb0c3f64d67807a475R6

And I updated the associated functions.

Reply:
Yea that is what I was thinking, cool.